scripts/update_readme_for_theme.py

```diff
@@ -63,15 +90,20 @@ def update_readme(theme):
         theme_docstring=getdoc(theme_cls),
         background_hex=hexstr_without_hash(style.background_color),
         highlight_hex=hexstr_without_hash(style.highlight_color),
-        colors_hex=foreground_colors.values(),
+        colors_hex=contrast_by_foreground_color(
+            color_cls, style.background_color
+        ).values(),
     )
 
     # Save the new README file
-    out = REPO / "a11y_pygments" / theme / "README.md"
+    out = outdir / "README.md"
     with open(out, "w") as f:
         logging.info(f"Updating {out}")
         f.write(result)
 
+    # Format the README file with prettier
+    subprocess.run(f"pre-commit run prettier --files {out}", shell=True)
```

Comment:
If we don't run Prettier on the Markdown file, then running this script will show all of the theme READMEs as needing to be staged in Git, but then when you run `git commit` and the pre-commit hook runs and updates the READMEs, there's nothing to commit! (because the changes were only Prettier differences)